tests/e2e/features/createSpace.feature

```diff
@@ -0,0 +1,31 @@
+Feature: Create a space
+    As a super user
+    I want to create a space
+    So that I can add recipes and share my recipes
+
+    Background:
+        Given the user has signed up with the following details
+            | username         | superuser |
+            | password         | super1234 |
+        And the user has logged in with username "superuser" and password "super1234"
+
+
+    Scenario: user creates a space
+        When the user creates a space "user's Space"
+        And the user should get success message "You have successfully created your own recipe space. Start by adding some recipes or invite other people to join you."
+        And user space "user's Space" should be visible on option menu
```

Comment:
Even though the user might be deleted after the step, logging out after the scenario might also be helpful. It's UI tests you might run into session problems. 
```suggestion
        And user space "user's Space" should be visible on option menu
        And the user "superuser" logs out
```

Reply:
logs out step have to close the session. but that has been done in the after-hook in this test setup (if I am not wrong). So, that should do the job

Reply:
If that's in after step then that's fine

Reply:
^ @KarunAtreya @nirajacharya2